SDAPackageWithVectorField/SDAWithVectorField/obstacle_map.py

```diff
@@ -0,0 +1,64 @@
+import numpy
+from math import atan2, cos, sin, pi, sqrt
+from SDAWithVectorField import *
+
+class ObstacleMap(object):
+	def __init__(self, drone_point, fly_zones):
+		self.obstacles = numpy.array([])
+		self.drone = Drone(drone_point, numpy.array([]))
+		self.fly_zones = FlightBoundariesContainer(fly_zones)
+		self.repulsive_forces = numpy.array([0,0,0])
+		self.attractive_force = numpy.array([0,0,0])
+		self.detection_radius = 0
+
+	def add_obstacle(self, obstacle):
+		if self.obstacles.size != 0:
+			final_obstacles = numpy.hstack([self.obstacles, obstacle])
+			self.obstacles = final_obstacles
+		else:
+			self.obstacles = numpy.array([obstacle])
+
+	def add_force(self):
+		for obstacle in self.obstacles:
+			if obstacle.get_type() == "moving":
+				force = VectorMath.get_force(self.drone.get_point(), obstacle.get_point())
+				self.repulsive_forces = numpy.sum([self.repulsive_forces, force], axis=0)
+
+	def set_attractive_force(self):
+		self.attractive_force = VectorMath.get_force(self.drone.point, self.drone.waypoint_holder.get_current_waypoint())
+
+	def reset_attractive_force(self):
+		self.attractive_force = numpy.array([0,0,0])
+
+	def reset_repulsive_forces(self):
+		self.repulsive_forces = numpy.array([0,0,0])
+
+	def get_avoidance_vector(self):
+		repulsive_forces = self.repulsive_forces
+		attractive_force = self.attractive_force
```

Comment:
Do you need to make these variables here or can you complete the calculation with the class variable?